modules/utxo-lib/src/bitgo/outputScripts.ts

```diff
@@ -63,3 +66,39 @@ export function createOutputScript2of3(pubkeys: Buffer[], scriptType: ScriptType
     witnessScript: witnessScript?.output,
   };
 }
+
+/**
+ * Creates and returns a taproot output script using the user and bitgo keys for the aggregate
+ * public key and a taptree containing a user+bitgo 2-of-2 script at the first depth level of the
+ * tree and user+backup and bitgo+backup 2-of-2 scripts one level deeper.
+ * @param pubkeys - a pubkey array containing the user key, backup key, and bitgo key in that order
+ * @returns {{scriptPubKey}}
+ */
+export function createTaprootScript2of3([userKey, backupKey, bitGoKey]: [Buffer, Buffer, Buffer]): SpendableScript {
```

Comment:
I wonder if we can unify this with `createOutputScript2Of3` - seems like we just have another case here that could be handled as part of that function

Reply:
I did originally try it that way but it felt a bit cluttered, I couldn't think of a better way of going about it besides just an`if (p2tr) then... else...` approach, at which point a separate function made sense. If I just added it to the `switch` statement then we'd be creating`script2of3` for nothing and creating a separate set of scripts used only by p2tr. I also thought it would be nice to be able to use this `[Buffer, Buffer, Buffer]` type with named array elements since the order of the keys matters and makes the code easier to read I think. Technically `createOutputScript2of3` could also use that type for its `pubkeys` parameter but it would've required a decent bit of refactoring to do.

However it might make sense to have `createTaprootscript2of3` called from within `createOutputScript2of3`, I'll explore that in a separate PR.

Reply:
for clarity I would prefer we used the named keys in `createOutputScript2of3` and supported `case 'p2tr'` there